lua/neogit/popups/help/init.lua

```diff
@@ -26,9 +26,7 @@ function M.create(env)
     p = p:action(cmd.keys, cmd.name, cmd.fn)
   end
 
-  p = p:env(env):build()
-
-  p:show()
```

Comment:
This is subtle (and something that predates me) but `:show()` _doesn't_ return the same `self` as `:build()`

`:show()` is a method on the `Buffer` class, whereas `:build()` is a method on the popup builder. So, this should be:

```lua
p = p:build()
p:show()
```

Reply:
Copy that. Fixed in the latest latest force push.

That's interesting that we have a builder pattern that inherits some other non-builder methods.